java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/appetizer/AppetizerService.java

```diff
@@ -1,182 +1,27 @@
 package com.my.FoodTruckApp.appetizer;
-
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.server.ResponseStatusException;
-
 import java.util.ArrayList;
-import java.util.Optional;
 
 @Service
 @RequiredArgsConstructor
 public class AppetizerService {
-    //    AppetizerModel appetizer1 = new AppetizerModel(1, "small", "spicy", "dinner", 4);
-//    AppetizerModel appetizer2 = new AppetizerModel(2, "medium", "sweet", "breakfast", 6);
-//    ArrayList<AppetizerModel> appetizers = new ArrayList<>(Arrays.asList(appetizer1, appetizer2));
-    private final AppetizerRepository appetizerRepository; //creating a NULL appetizerRepository, just a field
-    //final means cannot be changed, only instantiated once
-    //picked up by @RequiredArgsConstructor
-
-//    public AppetizerService(AppetizerRepository appetizerRepository) {
-//        //constructor
-//        this.appetizerRepository = appetizerRepository; //how it knows which appetizerRepository its using, orange is recieved, white is the class field
-//        //ASSIGNING what I recieved from the constructor
-//        //service now REQUIRES appetizerRepository
-//    } THIS IS THE ALL ARGS CONSTRUCTOR, NO LONGER NEEDED BECAUSE OF @AllArgsConstructor
-
-    //---------------------------------------------------------------------------------------------------------------------------------
+    private final AppetizerRepository appetizerRepository;
```

Comment:
super minor: can we add a new line before and after this dependency?

Reply:
Yes, I have changed added the new line